grakn-graql-shell/src/main/java/ai/grakn/graql/GraqlShell.java

```diff
@@ -253,38 +245,20 @@ public static void runShell(String[] args, String version, String historyFilenam
 
     private static String loadQuery(String filePath) throws IOException {
             List<String> lines = Files.readAllLines(Paths.get(filePath), StandardCharsets.UTF_8);
-            return lines.stream().collect(Collectors.joining("\n"));
+            return lines.stream().collect(joining("\n"));
     }
 
     private static void sendBatchRequest(String uriString, String graqlPath, String keyspace) throws IOException {
-        byte[] out = ("{\"path\": \"" + escapeJavaScript(graqlPath) + "\"}").getBytes(StandardCharsets.UTF_8);
+        LoaderClient loaderClient = new LoaderClient(keyspace, uriString);
```

Comment:
A common use case for the Loader API will be to push data in without caring that much about immediate consistency. With that in mind the rest API it hits should allow you to use a batch graph or a normal graph.